script.js

```diff
@@ -0,0 +1,28 @@
+const menu = document.querySelector(".menu");
+const menuItems = document.querySelectorAll(".menuItem");
+const hamburger = document.querySelector(".hamburger");
+const closeIcon = document.querySelector(".closeIcon");
+const menuIcon = document.querySelector(".menuIcon");
```

Comment:
Hello @Tufahel , you have indeed done a great job with the mobile menu feature and I have checked and most of the requirements have been met 💪🏻! However, since you are adding javaScript to this project, I suggest that you set up both JavaScript and HTML&CSS linters in the project repository. It will help you have quality code for the javaScript file too.

Reply:
Thank you for your review. I have solved the issue with javascript linters.